maturity-level-one/src/README.md

```diff
@@ -1,10 +1,7 @@
 # Practical API Guidelines - Maturity level One "Must have" - Code Samples
 ****************************************************************
-Code samples for maturity level One "Must have".
-****************************************************************
-
-These include:
-- URL NAMING
+This implementation includes:
+- URL NAMING conventions
```

Comment:
Let's improve formatting by using `#` or `##` for these sections

Reply:
updated